<!DOCTYPE HTML>
<html>

<head>
  <title>JUN CHEN</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="style/style.css" title="style" />
  <link rel="stylesheet" href="style/font-awesome.min.css"/>
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h3><a href="index.html">Welcome to the Chen Lab at Mayo Clinic!</a></h3>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li><a href="team.html">Team</a></li>
          <li  class="selected"><a href="research.html">Research</a></li>
          <li><a href="publication.html">Publication</a></li>
          <li><a href="software.html">Software</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </div>
    </div>
  </div>
    <div id="content_header"></div>
    <div id="site_content">
      <div id="content" style="width:880px">
        <!-- insert the page content here -->

        <p1><strong>Quantitative methods for microbiome and single-cell sequencing data</strong><br /></p1>
        <p>Focus on addressing the specific data characteristics of microbiome sequencing data which are phylogeny-constrained, zero-inflated and overdispersed high-dimensional count data in various statistical and computational problems, including testing the association between the microbiome and disease; microbiome subtype discovery; microbiome biomarker discovery; microbiome-based predictive models; and integrating microbiome data with other omics data. I am interested in developing statistical methods for single-cell sequencing data, including cell clustering, differential abundance analysis, differential expression analysis, and multimodal data integration.<br />
      </p>


      <p1><strong>Statistical methods for high-dimensional and compositional data analysis</strong><br /></p1>
      <p>Motivated by specific problems in genomic and microbiome data analysis, I am active in developing general statistical methodologies for high-dimensional and compositional data analysis. My research focuses on integrating auxiliary data, including prior biological knowledge and structural information, and on addressing compositional effects in various statistical tasks of high-dimensional data analysis, including multiple testing, regression analysis, canonical correlation analysis, and confounder adjustment.
    </p>

    <p1><strong>Quantitative methods for epigenetic and epigenomic data</strong><br /></p1>
    <p>I am also interested in developing statistical methods for epigenetic and epigenomic studies, including epigenome-wide association studies, multiple-trait expression quantitative trait loci (eQTL) studies, and integrative genomics. I am particularly interested in methods that integrate microbiome data with other omics data to better understand the biology of disease.
  </p>

  <p1><strong>Collaborative biomedical research</strong><br /></p1>
  <p>My collaborations span numerous diseases. I seeks to understand the microbiome etiology of many diseases by rigorous statistical procedures, including establishing disease-microbiome association, quantifying the microbiome's contribution in disease pathogenicity, identifying disease subtypes, and discovering particular microbiome features that correlate with various disease phenotypes.
</p>
      </div>
    </div>
</body>
</html>
