<!DOCTYPE HTML>
<html>

<head>
  <title>Jun Chen</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="style/style.css" title="style" />
  <link rel="stylesheet" href="style/font-awesome.min.css"/>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h3><a href="index.html">Welcome to the Chen Lab at Mayo Clinic!</a></h3>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li><a href="team.html">Team</a></li>
          <li><a href="research.html">Research</a></li>
          <li><a href="publication.html">Publication</a></li>

          <li class="selected"><a href="software.html">Software</a></li>
          <li><a href="contact.html">Contact</a></li>
        </ul>
      </div>
    </div>
    <div id="content_header"></div>
    <div id="site_content">
      <div id="content1">
        <!-- insert the page content here -->
        <div class="column">
          <div class="content2">


            <p1>
              <strong>MicrobiomeStat</strong><br /></p1>
              <p>A suite of methods for powerful and robust microbiome data analysis
              addressing zero-inflation, phylogenetic structure and compositional
              effects. The methods can be applied to the
              analysis of other (high-dimensional) compositional data arising from
              sequencing experiments.</p>
            <div class="field is-grouped is-grouped-multiline">
              <div class="control">
                <div class="tags has-addons">
                  <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
                  <a class="tag" href="https://cran.r-project.org/web/packages/MicrobiomeStat/index.html">CRAN</a>
                </div>
              </div>
              <div class="control">
                <div class="tags has-addons">
                  <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
                  <a class="tag" href="https://cran.r-project.org/web/packages/MicrobiomeStat/MicrobiomeStat.pdf">Documentation</a>
                </div>
              </div>
              <div class="control">
                <div class="tags has-addons">
                  <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
                  <a class="tag" href="https://arxiv.org/abs/2104.00242">Publication</a><br />
                </div>
              </div>
              </div>


              <p1>
                <strong><br />TDFDR</strong><br /></p1>
                <p>The package implements the two-dimensional false discovery rate control for powerful confounder adjustment in omics association analysis. The method is based on the idea that the confounder(s) usually affect part of the omics features, and thus adjusting the confounder(s) for ALL omics features will be over-adjustment, leading to reduced statistical power. The proposed procedure starts with performing the unadjusted analysis (first dimension - filtering) to narrow down the list of omics features that are more likely to be affected by either the confounder or the variable of interest or both. In the second dimension, we conduct confounder-adjusted analysis on these 'top' candidates, which are enriched in signals, to reduce multiple testing burden and increase the power. The method belongs to the general topic of using auxiliary data to increase the power of multiple testing, which has recently received tremendous research interest. In our case, the auxiliary data are the the unadjusted statistics, which could inform the probability of the null hypotheses being true. The difficulty here is to take into account the correlation between the auxiliary data (unadjusted statistics) and the main data (adjusted statistics). We provide a procedure that is theoretically guaranteed to control the false discovery rate while maximizing the power.
              </p>
              <div class="field is-grouped is-grouped-multiline">
                <div class="control">
                  <div class="tags has-addons">
                    <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
                    <a class="tag" href="https://github.com/jchen1981/TDFDR/">GitHub</a>
                  </div>
                </div>
                <div class="control">
                  <div class="tags has-addons">
                    <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
                    <a class="tag" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02418-8">Publication</a>
                  </div>
                </div>
              </div>


                <p1>
                  <strong><br />OrderShapeEM</strong><br /></p1>
                  <p>OrderShapeEM implements the optimal false discovery rate (FDR) control procedure with auxiliary information, particularly for prior ordering information. The framework is based on local FDR with hypothesis-specific null probability. The prior null proabilities are estimated using isotonic regression (PAVA algorithm) with respect to the prior ordering information. The inputs of our OrderShapeEM are simply P-values and their prior ordering.
                </p>
                <div class="field is-grouped is-grouped-multiline">
                  <div class="control">
                    <div class="tags has-addons">
                      <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
                      <a class="tag" href="https://github.com/jchen1981/OrderShapeEM">GitHub</a>
                    </div>
                  </div>
                  <div class="control">
                    <div class="tags has-addons">
                      <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
                      <a class="tag" href="https://arxiv.org/abs/2103.15311">Publication</a>
                    </div>
                  </div>
            </div>


            <p1>
              <strong><br />CAMT</strong><br /></p1>
              <p>The CAMT package implements two covariate adaptive multiple testing procedures (FDR and FWER) described in Covariate Adaptive False Discovery Rate Control with Applications to Omics-Wide Multiple Testing and Covariate Adaptive Family-wise Error Control with Applications to Genome-wide Association Studies. CAMT allows the prior null probability and/or the alternative distribution to depend on covariates. It is robust to model mis-specification and is computationally efficient. The package also contains functions for testing the informativeness of the covariates for multiple testing, and a comprehensive simulation function, which covers a wide range of settings.
            </p>
            <div class="field is-grouped is-grouped-multiline">
              <div class="control">
                <div class="tags has-addons">
                  <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
                  <a class="tag" href="https://github.com/jchen1981/CAMT">GitHub</a>
                </div>
              </div>
              <div class="control">
                <div class="tags has-addons">
                  <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
                  <a class="tag" href="https://arxiv.org/abs/1909.04811">Publication</a>
                </div>
              </div>
        </div>



        <p1>
          <strong><br />GUniFrac</strong><br /></p1>
          <p>A suite of methods for powerful and robust microbiome data analysis including data normalization, data simulation, community-level association testing and differential abundance analysis. It implements generalized UniFrac distances, Geometric Mean of Pairwise Ratios (GMPR) normalization, semiparametric data simulator, distance-based statistical methods, and feature-based statistical methods. The distance-based statistical methods include three extensions of PERMANOVA: (1) PERMANOVA using the Freedman-Lane permutation scheme, (2) PERMANOVA omnibus test using multiple matrices, and (3) analytical approach to approximating PERMANOVA p-value. Feature-based statistical methods include linear model-based methods for differential abundance analysis of zero-inflated high-dimensional compositional data.
        </p>
        <div class="field is-grouped is-grouped-multiline">
          <div class="control">
            <div class="tags has-addons">
              <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
              <a class="tag" href="https://cran.r-project.org/web/packages/GUniFrac/index.html">CRAN</a>
            </div>
          </div>
          <div class="control">
            <div class="tags has-addons">
              <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
              <a class="tag" href="https://cran.r-project.org/web/packages/GUniFrac/GUniFrac.pdf">Documentation</a>
            </div>
          </div>
    </div>



    <p1>
      <strong><br />GMPR</strong><br /></p1>
      <p>Calculate the normalizing factors for zeroinflated sequencing data such as microbiome sequencing data.
    </p>
    <div class="field is-grouped is-grouped-multiline">
      <div class="control">
        <div class="tags has-addons">
          <span class="tag is-dark"><i class="fa fa-code-fork" aria-hidden="true"></i>&nbsp; </span>
          <a class="tag" href="https://github.com/jchen1981/GMPR">GitHub</a>
        </div>
      </div>
      <div class="control">
        <div class="tags has-addons">
          <span class="tag is-dark"><i class="fa fa-book" aria-hidden="true"></i></span>
          <a class="tag" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5885979/">Publication</a>
        </div>
      </div>
</div>






          </div>
        </div>
        </div>

      </div>
    </div>
  </div>
</body>
</html>
